priv/emqx_web_hook.schema

```diff
@@ -5,36 +5,31 @@
   {datatype, string}
 ]}.
 
-{mapping, "web.hook.ssl", "emqx_web.hook.ssl", [
-  {default, off},
-  {datatype, flag}
-]}.
-
-{mapping, "web.hook.ssl.cafile", "emqx_web_hook.ssloptions", [
+{mapping, "web.hook.ssl.cafile", "emqx_web_hook.cafile", [
   {default, ""},
   {datatype, string}
 ]}.
 
-{mapping, "web.hook.ssl.certfile", "emqx_web_hook.ssloptions", [
+{mapping, "web.hook.ssl.certfile", "emqx_web_hook.certfile", [
```

Comment:
I think we need add this translation back:

```
{translation, "emqx_web_hook.ssloptions", fun(Conf) ->
     CA = cuttlefish:conf_get("web.hook.ssl.cafile", Conf),
     Cert = cuttlefish:conf_get("web.hook.ssl.certfile", Conf),
     Key = cuttlefish:conf_get("web.hook.ssl.keyfile", Conf),
     case ((Cert == "") or (Key == "")) of
          true -> [{cacertfile, CA}];
          _ -> [{cacertfile, CA}, {certfile, Cert}, {keyfile, Key}]
     end
end}.
```

1. In the v4.2.0-4.2.4,  these options has translation into `ssloptions`.
2. If we must change this data structure, we need to be compatible with the original `ssloptions` structure in 4.2.5 and 4.2.6. However, the `translation_env/1` function does not find compatibility with the 4.2.0-4 structure. So Hot Upgrades should not work with SSL